app/models/journaled/delivery.rb

```diff
@@ -60,10 +60,16 @@ def legacy_credentials_hash_if_present
     end
   end
 
+  def sts_client_config
+    {
+      region: ENV.fetch('AWS_DEFAULT_REGION', DEFAULT_REGION)
```

Comment:
would it make sense to just add region to `legacy_credentials_hash_if_present`

Reply:
legacy creds are outside of AWS's conventional ENV var names, AWS_DEFAULT_REGION is actually respected by AWS clients directly, I believe?

Reply:
i refactored this, didn't see y'all's comments! it turns out the bug was on the kinesis client config, so check out https://github.com/Betterment/journaled/pull/5/commits/29b8f957a032b1f15fc96831ef218209024a58cb

Reply:
update! the bug is on both! both clients need the region to be specified. and the sts client uses the legacy credentials, but the kinesis client uses either the Assume role creds (which _may_ include the legacy creds) or JUST the legacy creds. it gets a bit chicken-or-egg-y, which is why both I decided to initialize both clients explicitly with whichever creds they need plus the region.

Reply:
sgtm!